backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/InteropCallConvertors.kt

```diff
@@ -56,6 +57,14 @@ private class InteropCallContext(
 
     fun IrType.isStoredInMemoryDirectly(): Boolean =
             isPrimitiveType() || isUnsigned() || isVector()
+
+    fun IrType.isSupportedReference(): Boolean = isObjCObjectType()
+            || isNullableString()
```

Comment:
Why nullable?

Reply:
Because we generate field of nullable type for `NSString*`. 
Oh, I see that there is a `IrType.isStringClassType()` extension which is more flexible.

Reply:
> Because we generate field of nullable type for `NSString*`.

Which is probably not true if the field is declared as non-nullable:
```obj-c
NSString* _Nonnull s;
```

Reply:
Yeah, right. Anyway, this code is superseded by 6d13354.

Reply:
This case should be checked by tests too (same for https://github.com/JetBrains/kotlin-native/pull/4178#discussion_r427171368 and https://github.com/JetBrains/kotlin-native/pull/4178#discussion_r427854784).